Cargo.toml

```diff
@@ -180,21 +180,45 @@ members = [
 ]
 
 [profile.dev]
-opt-level = 0  # Controls the --opt-level the compiler builds with
-debug = true   # Controls whether the compiler passes `-g`
+opt-level = 0
+debug = false
 codegen-units = 4
+lto = false
+incremental = true
+panic = 'unwind'
+debug-assertions = false
+overflow-checks = false
+rpath = false
 
 # The release profile, used for `cargo build --release`
 [profile.release]
-lto = true
 opt-level = 3
 debug = true
-# TODO: remove this once rust-lang/rust#50199 and rust-lang/rust#53833 are resolved.
-codegen-units = 1
+codegen-units = 1 # TODO: remove this once rust-lang/rust#50199 and rust-lang/rust#53833 are resolved.
+lto = true
+incremental = false
+panic = 'unwind'
```

Comment:
How about `abort`? It may save some time to link.

Reply:
I have done that in a different PR https://github.com/tikv/tikv/pull/4639. If I recall the rationale was that it would be easier to review and land that PR than this one. They will have merge conflicts.